packages/terra-form-select/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed incorrect role for upcoming axe-core version bump
```

Comment:
i think we should also mention about the aria tag changes also here.

Reply:
Updated here